docs/manuals/source/Appendix/Checkpoints.rst

```diff
@@ -0,0 +1,18 @@
+.. _checkpoints-chapter:
+
+Checkpoints
+===========
+
```

Comment:
Usually, the information about files that were backed up in a job are stored into the database a the end of a successful job.
This results in the fact, that failed or cancelled jobs have no file information.
The data from this jobs cannot be recovered via the usual restore procedure.
Especially for very long running jobs with many files, this fact is unsatisfactory. 

To avoid this problem, the Checkpoints feature was introduced.